Make.inc

```diff
@@ -991,7 +991,7 @@ USE_BINARYBUILDER ?= 0
 endif
 
 # This is the set of projects that BinaryBuilder dependencies are hooked up for.
-BB_PROJECTS := OPENBLAS LLVM SUITESPARSE OPENLIBM GMP MBEDTLS LIBSSH2 MPFR CURL LIBGIT2 PCRE LIBUV LIBUNWIND DSFMT OBJCONV
+BB_PROJECTS := OPENBLAS LLVM SUITESPARSE OPENLIBM GMP MBEDTLS LIBSSH2 MPFR CURL LIBGIT2 PCRE LIBUNWIND DSFMT OBJCONV
```

Comment:
You shouldn't never need to do this; if you don't want to use BB for a dep, you can set `override USE_BINARYBUILDER_LIBUV := 0` in your `Make.user`.